### ISSUE TITLE
How to get model_cancer ?

### ISSUE THREAD

**Issue by EmmaSwan13** [coffee19850519/single_cell_spatial_image#8]
Hi Fei,
Your group's work is brilliant and I've learned a lot. I have a question and hope you can help me with the answer. I replicated your code published on Github. But I was confused how to train and get the model_cancer. In your article, you mention that the segmentation model is a supervised convolutional neural network model which means the annotation labels of the regions are needed. However, there is no annotation file in the folder "cancer". Could you tell me how to get the annotation? It will help me a lot. Thank you!

Best wishes,
Yu

**Comment by coffee19850519** [comment]
Hi Yu,
Thanks for your interest.
Actually, our model_cancer was trained on public annotated brain data. Indeed, as you found there is no annotated cancer data, so we just tested the pre-trained model on cancer data and investigated its generalizability.  And also our predicted results on cancer tissue were validated by a pathologist rather than annotations in our article.
However, if you found any new annotated data, you can follow [this](https://github.com/coffee19850519/single_cell_spatial_image#function-5-customize-segmentation-model-gpu-required) to further tune the segmentation model, which is expected to achieve better performance.  

Fei

**Comment by EmmaSwan13** [comment]
Hi Fei,
Thanks for taking the time out of your busy schedule to reply. I will try it later. Hope that I can also get your help if I have other trouble.

Best wishes,
Yu
